packages/components/credentials/UpstashVectorApi.credential.ts

```diff
@@ -0,0 +1,29 @@
+import { INodeParams, INodeCredential } from '../src/Interface'
+
+class UpstashVectorApi implements INodeCredential {
+	label: string
+	name: string
+	version: number
+	description: string
+	inputs: INodeParams[]
+
+	constructor() {
+		this.label = 'Upstash Vector API'
+		this.name = 'upstashVectorApi'
+		this.version = 1.0
+		this.inputs = [
+			{
+				label: 'UPSTASH_VECTOR_REST_URL',
```

Comment:
Can we change the label to `Upstash Vector REST URL`. Same with `Upstash Vector REST Token`